.github/workflows/build_wheels.yml

```
@@ -20,9 +20,10 @@ jobs:
         uses: pypa/cibuildwheel@v2.16.2
         env:
           CIBW_ARCHS_MACOS: x86_64 arm64
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
           path: ./wheelhouse/*.whl
+          name: wheels
```

```suggestion
          name: wheels-${{ matrix.os }}
```

Just saw that this is a matrix